src/sentry/api/endpoints/organization_member_commits.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+
+from sentry.api.bases import OrganizationMemberEndpoint
+from sentry.api.paginator import OffsetPaginator
+from sentry.api.serializers import serialize, CommitWithReleaseSerializer
+from sentry.db.models.query import in_iexact
+from sentry.models import Commit, CommitAuthor, UserEmail
+
+
+class OrganizationMemberCommitsEndpoint(OrganizationMemberEndpoint):
+    def get(self, request, organization, member):
+        email_list = list(UserEmail.objects.filter(
+            user=member.user_id,
+            is_verified=True,
+        ).values_list('email', flat=True))
+        if email_list:
+            queryset = Commit.objects.filter(
+                organization_id=organization.id,
+                author__in=CommitAuthor.objects.filter(
+                    in_iexact('email', email_list),
```

Comment:
Not a blocker or anything, but we should probably either put a functional index here, or make it citext.

Just noting because this will just be filtering over all authors within an org.

The best index we have here currently is `"sentry_commitauthor_organization_id_5656e6a6baa5f6c_uniq" UNIQUE CONSTRAINT, btree (organization_id, email)` and it'll only scan the organization_id half. But the data set here shouldn't be too large to cause a problem.